### ISSUE TITLE
Get colored icons in i3blocks

### ISSUE THREAD

**Issue by enanajmain** [LukeSmithxyz/voidrice#159]
You have some colored icons for your i3blocks. Where do I get it?

I have copied some from your config file and it works ok in i3blocks but I can't see it in my terminal st. Do I need to install a font for my st terminal?

**Comment by LukeSmithxyz** [comment]
Yes, these are all just based on what emoji fonts you have. The emoji font in my status bar is `ttf-emojione` from the AUR. My monospace font used in `st` is `ttf-inconsola`. I also have `ttf-symbola` (AUR) installed as well, although I forget if this supplies emojis as I might just use it for other unicode characters.
